src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1028,9 +1029,22 @@
                     Condition="'%(PackageFile.IsSourceCodeFile)'!='true'" />
     </ItemGroup>
   </Target>
+  
+  <Target Name="GetPackageReport"
```

Comment:
It's triggered by ValidatePackage.  It can also be triggered on its own.  EG: by adding to dir.traversal.targets and calling it on packages.builds.
I'll also add it to BuildDependsOn to ensure it runs even if SkipValidatePackage is set.